configs/components/rubygem-childprocess.rb

```diff
@@ -1,6 +1,6 @@
 component "rubygem-childprocess" do |pkg, settings, platform|
-  pkg.version "4.1.0"
-  pkg.sha256sum '3616ce99ccb242361ce7f2b19bf9ff3e6bc1d98b927c7edc29af8ca617ba6cd3'
+  pkg.version "5.0.0"
+  pkg.sha256sum '0746b7ab1d6c68156e64a3767631d7124121516192c0492929a7f0af7310d835'
```

Comment:
Can you mirror this gem in the release-public channel? Looks like Craig last mirrored this gem in Feb 22, 2023

Reply:
Done
![Screenshot 2024-05-16 at 8 48 39 AM](https://github.com/puppetlabs/puppet-runtime/assets/21069923/f5096984-4460-47a4-a0f8-f69dfda86a33)